src/main/kotlin/dev/stalla/builder/validating/ValidatingRssCategoryBuilder.kt

```diff
@@ -23,6 +23,9 @@ internal class ValidatingRssCategoryBuilder : RssCategoryBuilder {
             return null
         }
 
-        return RssCategory(categoryValue, domain)
+        return RssCategory(
+            name = categoryValue,
+            domain = domain
+        )
```

Comment:
Also switched to named arguments in all builders. First for consistency, and second like in this case we sometimes have diverging names between the caller variable and the callee parameter.